jwst/superbias/bias_sub.py

```diff
@@ -42,33 +35,32 @@ def do_correction(input_model, bias_model):
     # Subtract the bias ref image from the science data
     output_model = subtract_bias(input_model, bias_model)
 
-    output_model.meta.cal_step.superbias = 'COMPLETE'
+    output_model.meta.cal_step.superbias = "COMPLETE"
 
     return output_model
 
 
 def subtract_bias(output, bias):
     """
-    Subtracts a superbias image from a science data set, subtracting the
-    superbias from each group of each integration in the science data.
+    Subtract a superbias image from a science data set.
+
+    Subtract the superbias from each group of each integration in the science data.
+
     The DQ flags in the bias reference image are propagated into the science
-    data pixeldq array. The error array is unchanged.
+    data pixel dq array. The error array is unchanged.
```

Comment:
I recommend capitalizing "DQ" here, since it's now changed to an acronym, instead of a variable name.